[2023-11-02T10:10:13.421+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: test_dag_tutorial.create_postgres_table scheduled__2020-01-09T00:00:00+00:00 [queued]>
[2023-11-02T10:10:13.552+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: test_dag_tutorial.create_postgres_table scheduled__2020-01-09T00:00:00+00:00 [queued]>
[2023-11-02T10:10:13.552+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-11-02T10:10:13.553+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 6
[2023-11-02T10:10:13.554+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-11-02T10:10:14.143+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2020-01-09 00:00:00+00:00
[2023-11-02T10:10:14.165+0000] {standard_task_runner.py:55} INFO - Started process 1152 to run task
[2023-11-02T10:10:14.182+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'test_dag_tutorial', 'create_postgres_table', 'scheduled__2020-01-09T00:00:00+00:00', '--job-id', '75', '--raw', '--subdir', 'DAGS_FOLDER/test_dag_tutorial.py', '--cfg-path', '/tmp/tmp8cf9x5m6']
[2023-11-02T10:10:14.192+0000] {standard_task_runner.py:83} INFO - Job 75: Subtask create_postgres_table
[2023-11-02T10:10:14.504+0000] {task_command.py:376} INFO - Running <TaskInstance: test_dag_tutorial.create_postgres_table scheduled__2020-01-09T00:00:00+00:00 [running]> on host b33dd0bd4b35
[2023-11-02T10:10:14.822+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=tetiana
AIRFLOW_CTX_DAG_ID=test_dag_tutorial
AIRFLOW_CTX_TASK_ID=create_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2020-01-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-09T00:00:00+00:00
[2023-11-02T10:10:14.865+0000] {base.py:71} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-11-02T10:10:14.873+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/operators/postgres.py", line 94, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 288, in run
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/hooks/postgres.py", line 115, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5432 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?

[2023-11-02T10:10:14.936+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=test_dag_tutorial, task_id=create_postgres_table, execution_date=20200109T000000, start_date=20231102T101013, end_date=20231102T101014
[2023-11-02T10:10:14.972+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 75 for task create_postgres_table (connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5432 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
; 1152)
[2023-11-02T10:10:15.026+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-11-02T10:10:15.128+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2023-11-02T10:20:00.233+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: test_dag_tutorial.create_postgres_table scheduled__2020-01-09T00:00:00+00:00 [queued]>
[2023-11-02T10:20:00.277+0000] {taskinstance.py:1165} INFO - Dependencies all met for <TaskInstance: test_dag_tutorial.create_postgres_table scheduled__2020-01-09T00:00:00+00:00 [queued]>
[2023-11-02T10:20:00.278+0000] {taskinstance.py:1362} INFO - 
--------------------------------------------------------------------------------
[2023-11-02T10:20:00.279+0000] {taskinstance.py:1363} INFO - Starting attempt 2 of 6
[2023-11-02T10:20:00.279+0000] {taskinstance.py:1364} INFO - 
--------------------------------------------------------------------------------
[2023-11-02T10:20:00.362+0000] {taskinstance.py:1383} INFO - Executing <Task(PostgresOperator): create_postgres_table> on 2020-01-09 00:00:00+00:00
[2023-11-02T10:20:00.374+0000] {standard_task_runner.py:55} INFO - Started process 1663 to run task
[2023-11-02T10:20:00.383+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'test_dag_tutorial', 'create_postgres_table', 'scheduled__2020-01-09T00:00:00+00:00', '--job-id', '112', '--raw', '--subdir', 'DAGS_FOLDER/test_dag_tutorial.py', '--cfg-path', '/tmp/tmpt_g8jo52']
[2023-11-02T10:20:00.399+0000] {standard_task_runner.py:83} INFO - Job 112: Subtask create_postgres_table
[2023-11-02T10:20:00.728+0000] {task_command.py:376} INFO - Running <TaskInstance: test_dag_tutorial.create_postgres_table scheduled__2020-01-09T00:00:00+00:00 [running]> on host b33dd0bd4b35
[2023-11-02T10:20:01.347+0000] {taskinstance.py:1592} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=tetiana
AIRFLOW_CTX_DAG_ID=test_dag_tutorial
AIRFLOW_CTX_TASK_ID=create_postgres_table
AIRFLOW_CTX_EXECUTION_DATE=2020-01-09T00:00:00+00:00
AIRFLOW_CTX_TRY_NUMBER=2
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2020-01-09T00:00:00+00:00
[2023-11-02T10:20:01.460+0000] {base.py:71} INFO - Using connection ID 'postgres_localhost' for task execution.
[2023-11-02T10:20:01.509+0000] {taskinstance.py:1851} ERROR - Task failed with exception
Traceback (most recent call last):
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/operators/postgres.py", line 94, in execute
    self.hook.run(self.sql, self.autocommit, parameters=self.parameters)
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/common/sql/hooks/sql.py", line 288, in run
    with closing(self.get_conn()) as conn:
  File "/home/airflow/.local/lib/python3.7/site-packages/airflow/providers/postgres/hooks/postgres.py", line 115, in get_conn
    self.conn = psycopg2.connect(**conn_args)
  File "/home/airflow/.local/lib/python3.7/site-packages/psycopg2/__init__.py", line 122, in connect
    conn = _connect(dsn, connection_factory=connection_factory, **kwasync)
psycopg2.OperationalError: connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5432 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?

[2023-11-02T10:20:01.599+0000] {taskinstance.py:1406} INFO - Marking task as UP_FOR_RETRY. dag_id=test_dag_tutorial, task_id=create_postgres_table, execution_date=20200109T000000, start_date=20231102T102000, end_date=20231102T102001
[2023-11-02T10:20:01.754+0000] {standard_task_runner.py:105} ERROR - Failed to execute job 112 for task create_postgres_table (connection to server at "localhost" (127.0.0.1), port 5432 failed: Connection refused
	Is the server running on that host and accepting TCP/IP connections?
connection to server at "localhost" (::1), port 5432 failed: Cannot assign requested address
	Is the server running on that host and accepting TCP/IP connections?
; 1663)
[2023-11-02T10:20:01.864+0000] {local_task_job.py:164} INFO - Task exited with return code 1
[2023-11-02T10:20:02.397+0000] {local_task_job.py:273} INFO - 0 downstream tasks scheduled from follow-on schedule check
